kolibri/core/assets/src/api_resource.js

```diff
@@ -425,11 +410,10 @@ class Resource {
    * @param {String} id - The primary key of the Model instance.
    * @returns {Model} - Returns a Model instance.
    */
-  getModel(id, data = {}) {
```

Comment:
@indirectlylit the problem happens with this new definition of getModel. Change it to the original and it works. Currently investigating..